packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -214,21 +215,33 @@ const DefaultSummaryConfiguration: ISummaryConfiguration = {
  * Options for container runtime.
  */
 export interface IContainerRuntimeOptions {
-    // Flag that will generate summaries if connected to a service that supports them.
-    // This defaults to true and must be explicitly set to false to disable.
+    /**
+     * Flag that will generate summaries if connected to a service that supports them.
+     * This defaults to true and must be explicitly set to false to disable.
+     */
     generateSummaries?: boolean;
 
-    // Delay before first attempt to spawn summarizing container
+    /* Delay before first attempt to spawn summarizing container. */
     initialSummarizerDelayMs?: number;
 
-    // Flag that will disable garbage collection if set to true.
+    /* Flag that will disable garbage collection if set to true. */
     disableGC?: boolean;
```

Comment:
can we put all the gc options into a sub-object, rather than having them all directly on IContainerRuntimeOptions. seems to be sprawling a bit

Reply:
I broke into `summaryOptions` and `gcOptions`. Let me know what you think.